inference-engine/src/mkldnn_plugin/nodes/mkldnn_roi_pooling_node.cpp

```diff
@@ -676,35 +756,90 @@ void MKLDNNROIPoolingNode::execute() {
                     }
                 }
             }
+        });
+    }
 
-            if (roi_pooling_kernel) {
-                (*roi_pooling_kernel)(&arg);
-            }
-        }
-    });
+private:
+    jit_roi_pooling_params jpp;
+};
+
+std::shared_ptr<MKLDNNROIPoolingNode::ROIPoolingExecutor> MKLDNNROIPoolingNode::ROIPoolingExecutor::createROIPoolingNewExecutor(
+    const InferenceEngine::Precision& prc,
+    const jit_roi_pooling_params& jpp) {
```

Comment:
you already have precision inside jpp. Seems line can be reused here

Reply:
Corrected, thank you!